variables.tf

```diff
@@ -6,33 +6,41 @@ variable "location" {
 variable "env" {
   type = string
 }
+variable "product" {
+  type = string
+}
 variable "tags" {
   type        = map(string)
   description = "Runbook Tags"
 }
 ## Azure Automation
-variable "automation_account_name" {
-  type        = string
-  description = "automation account name"
+variable "auto_acc_runbooks" {
+  default = []
 }
-variable "publish_content_link" {
-  type        = string
-  description = "source of ps1 script"
+variable "resource_group_id" {
```

Comment:
Do we need this?

Reply:
no. this was a holdover from the previous. removed

Reply:
removed